tests/Backend/CommonPart1/TableExporterTest.php

```diff
@@ -284,24 +290,24 @@ public function tableExportData(): array
             [[self::BACKEND_REDSHIFT, self::BACKEND_SNOWFLAKE, self::BACKEND_SYNAPSE, self::BACKEND_EXASOL, self::BACKEND_TERADATA], new CsvFile($filesBasePath . 'numbers.csv'), 'numbers.csv'],
             [[self::BACKEND_REDSHIFT, self::BACKEND_SNOWFLAKE, self::BACKEND_SYNAPSE, self::BACKEND_EXASOL], new CsvFile($filesBasePath . 'numbers.csv'), 'numbers.two-cols.csv', ['columns' => ['0', '45']]],
             // TODO add TD ^^
 
             // tests the redshift data too long bug https://github.com/keboola/connection/issues/412
             // TD skipped because of TD limit 10666 chars
-            [[self::BACKEND_REDSHIFT, self::BACKEND_SNOWFLAKE, self::BACKEND_EXASOL], new CsvFile($filesBasePath . 'languages.64k.csv'), 'languages.64k.csv'],
-            [[self::BACKEND_REDSHIFT, self::BACKEND_SNOWFLAKE, self::BACKEND_EXASOL], new CsvFile($filesBasePath . 'languages.64k.csv'), 'languages.64k.csv',  ['gzip' => true]],
-            [[self::BACKEND_REDSHIFT, self::BACKEND_SNOWFLAKE, self::BACKEND_EXASOL], new CsvFile($filesBasePath . '64K.csv'), '64K.csv'],
-            [[self::BACKEND_REDSHIFT, self::BACKEND_SNOWFLAKE, self::BACKEND_EXASOL], new CsvFile($filesBasePath . '64K.csv'), '64K.csv',  ['gzip' => true]],
-
-            [[self::BACKEND_REDSHIFT], new CsvFile($filesBasePath . 'escaping.csv'), 'escaping.standard.out.csv', ['gzip' => true]],
-            [[self::BACKEND_REDSHIFT], new CsvFile($filesBasePath . 'numbers.csv'), 'numbers.csv', ['gzip' => true]],
-            [[self::BACKEND_REDSHIFT], new CsvFile($filesBasePath . 'numbers.csv'), 'numbers.two-cols.csv', ['gzip' => true, 'columns' => ['0', '45']]],
+            [[self::BACKEND_REDSHIFT, self::BACKEND_SNOWFLAKE, self::BACKEND_EXASOL, self::BACKEND_BIGQUERY], new CsvFile($filesBasePath . 'languages.64k.csv'), 'languages.64k.csv'],
+            [[self::BACKEND_REDSHIFT, self::BACKEND_SNOWFLAKE, self::BACKEND_EXASOL, self::BACKEND_BIGQUERY], new CsvFile($filesBasePath . 'languages.64k.csv'), 'languages.64k.csv',  ['gzip' => true]],
+            [[self::BACKEND_REDSHIFT, self::BACKEND_SNOWFLAKE, self::BACKEND_EXASOL, self::BACKEND_BIGQUERY], new CsvFile($filesBasePath . '64K.csv'), '64K.csv'],
+            [[self::BACKEND_REDSHIFT, self::BACKEND_SNOWFLAKE, self::BACKEND_EXASOL, self::BACKEND_BIGQUERY], new CsvFile($filesBasePath . '64K.csv'), '64K.csv',  ['gzip' => true]],
+
+            [[self::BACKEND_REDSHIFT, self::BACKEND_BIGQUERY], new CsvFile($filesBasePath . 'escaping.csv'), 'escaping.standard.out.csv', ['gzip' => true]],
+            [[self::BACKEND_REDSHIFT, self::BACKEND_BIGQUERY], new CsvFile($filesBasePath . 'numbers.csv'), 'numbers.csv', ['gzip' => true]],
+            [[self::BACKEND_REDSHIFT, self::BACKEND_BIGQUERY], new CsvFile($filesBasePath . 'numbers.csv'), 'numbers.two-cols.csv', ['gzip' => true, 'columns' => ['0', '45']]],
         ];
     }
 }
```

Comment:
Tady to chybý dořešit.
Test padá protože export z BQ nedělá value enclosure, takže místo "123" vyexportuje BQ 123.
Nejlepší mě přijde asi udělat vlastní Assertní csv. Prostě udělat BigQueryTableExport a udělat overide tady toho provideru kde budů jiné expectation CSV pro BQ.